float32[]  marker_id
float32[]  center_values
